_data/categories/meshes.yml

```diff
@@ -69,6 +59,27 @@ Open Source. From Buoyant. Written in Rust and Go."
   multi-cluster: "?"
   multi-tenant: "?"
 
+- name: Mesher
+  desc: "A service mesh implementation based on go chassis. One big advantage of Mesher is it is able to work with go-chassis in same service mesh control plane like Istio. Without Istio they can work together with ServiceComb Service center and running on any infrastructure(docker, VM, baremetal).
```

Comment:
Even thought the project's README isn't (https://github.com/go-chassis/go-chassis), let's be consistent with how we reference go-chassis. There are two different grammatical styles used in these couple sentences.

Let's avoid opinions, adjectives and comparisons/analogies in the description of the projects in attempt to provide unbiased depictions focused on facts. As examples, let's avoid "big advantage", "like Istio", "easily".

"docker" isn't an infrastructure.

Let's do what we can to cut down the length of descriptions here given that they are shown in tooltips - <a href="https://ibb.co/YDSgG1d"><img src="https://i.ibb.co/YDSgG1d/Screen-Shot-2019-01-17-at-6-16-09-AM.png" alt="Screen-Shot-2019-01-17-at-6-16-09-AM" border="0"></a>

Reply:
"A service mesh implementation based on go-chassis, Mesher can work with go-chassis in same service mesh control and can work with ServiceComb Service center running on any infrastructure. Mesher supports both linux and windows OS"

Thank you  Lee good Insights. Does the above condensed statement seem fair and unbiased.

Reply:
@leecalcote  this is the updated version . "Mesher is service mesh implementation based on go-chassis which can work together with ServiceComb Service center running on any infrastructure. Mesher can work with go-chassis in same service mesh control plane and it supports both linux and windows OS"